applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -251,6 +368,13 @@ public static ConfigurationAsyncClientBuilder builder() {
      * Fetches the configuration settings that match the {@code options}. If {@code options} is {@code null}, then all
      * the {@link ConfigurationSetting configuration settings} are fetched with their current values.
      *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <pre>
+     * client.listSettings(new SettingSelector().key("prodDBConnection")).subscribe(setting -&gt; {
```

Comment:
When you move the setting selector inline like this, in my opinion you should move the `.subscribe` to the next line.